models/PyCryptoBot.py

```diff
@@ -366,16 +371,16 @@ def getTime(self):
             return ''
 
     def isLive(self):
```

Comment:
what about adding the return type here (and other places), e.g.:
```python
def isLive(self) -> bool:
```

Reply:
Done